devito/types/basic.py

```diff
@@ -80,6 +80,11 @@ class Basic(object):
     is_SparseFunction = False
     is_PrecomputedSparseFunction = False
     is_PrecomputedSparseTimeFunction = False
+    is_TimeDependent = False
```

Comment:
This should not be glued with the `is_Sparse...` group

Reply:
Separated